theano/tensor/nnet/nnet.py

```diff
@@ -2448,6 +2448,31 @@ def elu(x, alpha=1):
     return tensor.switch(x > 0, x, alpha * tensor.expm1(x))
 
 
+def selu(x):
+    """Compute the element-wise Scaled Exponential Linear unit.
+
+    .. versionadded:: 0.9.0
+
+    Parameters
+    ----------
+    x : symbolic tensor
+        Tensor to compute the activation function for.
+
+    Returns
+    -------
+    symbolic tensor
+        Element-wise scaled exponential linear activation function applied to `x`.
+
+    References
+    ----------
+    .. [1] Klambauer G, Unterthiner T, Mayr A, Hochreiter S.
```

Comment:
This line see to raise an error in the doc gen:

/home/travis/build/Theano/Theano/theano/tensor/nnet/nnet.py:docstring of theano.tensor.nnet.selu:13: WARNING: Duplicate explicit target name: "1".

Maybe you can just remove the [1]?